python/sglang/srt/models/llama.py

```diff
@@ -400,6 +400,9 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
                 # Skip loading extra bias for GPTQ models.
                 if name.endswith(".bias") and name not in params_dict:
                     continue
+                # Skip loading kv_scale from ckpts towards new design.
```

Comment:
When it comes to e4m3, kv scale is needed. What is the reason for skipping here?

Reply:
current code can't load `.kv_scale` in checkpoint.
![image](https://github.com/user-attachments/assets/bbf1195d-bc0b-4460-8d31-41cd4bb13a73)

this is to make use of `amd/Meta-Llama-3.1-405B-Instruct-FP8-KV` first, we will address e4m3 kv cache scaling separately.

Reply:
Maybe we can complete it in a complete PR? Currently, this merge doesn't actually work for users.

Reply:
We will add scaled e4m3 support, yes as a complete feature.
My motive here is to unblock some use cases without harm 😄

Reply:
ok